### ISSUE TITLE
Can we stream tweets from a particular user?

### ISSUE THREAD

**Issue by mooningnow** [linvi/tweetinvi#846]
AddFollow returns the tweets where is user is mentioned or tagged.  I am interested only in the tweets tweeted by the user.

**Comment by Scobiform** [comment]
Addfollow should return the user tweets... post your stream code.

**Comment by mooningnow** [comment]
I am getting the user's tweets but along with that I am also getting all the spam tweets where the user's screen name is tagged/replied to, etc.

**Comment by Scobiform** [comment]
Skip Retweets and so on...

**Comment by mooningnow** [comment]
I could skip those programmatically after received the tweet. My question is can that be skipped in the user stream itself?

**Comment by jdahan91** [comment]
> My question is can that be skipped in the user stream itself?

No, this cannot be done unless linvi implements such thing which i believe will not because he wants to give the programmer full control over the result.

**Comment by linvi** [comment]
@jdahan91 is correct. This type of logic is "advanced" and therefore up to the user to implement it.
The reason is that such rules are hard to maintain over time as Twitter might decide to change the rules and it becomes hard to follow.
